strawberryfields/apps/qchem/dynamics.py

```diff
@@ -212,3 +215,36 @@ def sample_fock(
         s.append(eng.run(prog).samples[0].tolist())
 
     return s
+
+
+def prob(samples: list, excited_state: list) -> float:
+
+    r"""Generate probability of observing a Fock state.
```

Comment:
```suggestion
    r"""Estimate probability of observing an excited state.
    
    The probability is estimated by calculating the relative frequency of the excited
    state among the samples.
```
Just felt it could do with an extra 1 line explanation.